docs/constants/navigation.js

```diff
@@ -36,7 +36,7 @@ const sections = [
     reference: [
       'EAS Build',
       'Creating your first build',
-      'Configuration with eas.json',
+      'EAS Build - Configuration with eas.json',
```

Comment:
```suggestion
      'Configuring EAS Build with eas.json',
```

Imo this is a little better of a page title- and fewer characters. And then would say the same change to the EAS Submit title